### DISCUSSION TITLE
How to find an Issue to Solve?

### DISCUSSION THREAD

**Discussion by saikumarb12345** [EddieHubCommunity/support#7288] [First-Timers] [answered]
How to find an Issues to solve that and (which types of issues should I reply)?
I hope someone will give the answer for this........

**Comment by Panquesito7** [comment, marked as answer]
Great question! 👍
You can afford to these resources to find (good first) issues easier:

1. EddieHub issue finder: https://finder.eddiehub.io/
2. GitHub Explore page: https://github.com/explore
3. The OSWC (Open-source welcome committee): https://www.oswc.is/search-projects
4. All EddieHub projects that have lots of things that can be worked on: https://github.com/EddieHubCommunity

I hope that helped. Let us know if you need any more help. 🙂
